encord/orm/dataset.py

```diff
@@ -944,18 +944,53 @@ class ImagesDataFetchOptions:
 
 class LongPollingStatus(str, Enum):
     PENDING = "PENDING"
+    """Job will automatically start soon (waiting in queue) or already started processing."""
+
     DONE = "DONE"
+    """
+    Job is finished successfully (possibly with errors if `ignore_errors=True`)
+    If `ignore_errors=False` was specified in :meth:`encord.dataset.Dataset.add_private_data_to_dataset_start`
+    , job will have DONE status only if there were no errors.
+    If `ignore_errors=True` was specified in :meth:`encord.dataset.Dataset.add_private_data_to_dataset_start`
+    , job will always have `DONE` status at some point (after finished processing), job cannot have `ERROR`
```

Comment:
"job will always show the status 'DONE' once complete and will never show 'ERROR'."